### PR TITLE
refactor: Refactor `markdown.glimmer.codeblock.mjs`

### PR DESCRIPTION
**Pull request by spuxx1701**
Tiny refactor as suggested by @evoactivity 🥳

### CODE REVIEW

File: syntaxes/src/markdown.glimmer.codeblock.mjs [hunk 1]
```diff
@@ -11,6 +11,42 @@ function regexes(extension) {
   };
 }
 
+function codeblock(extension) {
```
Comment (lifeart) [inline syntaxes/src/markdown.glimmer.codeblock.mjs:14]: good job! could is generate command has been executed (to ensure we have same output)?

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -0,0 +1,2521 @@
+lockfileVersion: '6.0'
```
Comment (lifeart) [inline pnpm-lock.yaml:1]: seems this file not need :)

Comment (spuxx1701) [reply pnpm-lock.yaml]: oopsie daisy

File: syntaxes/src/markdown.glimmer.codeblock.mjs (hunk 1 above)
Comment (spuxx1701) [reply syntaxes/src/markdown.glimmer.codeblock.mjs:14]: Sorry, I don't quite understand. :smile:

Comment (evoactivity) [reply syntaxes/src/markdown.glimmer.codeblock.mjs:1]: Can you run the build script and commit the json files, just to ensure the output is the same :)
The josn files are used directly from this repo in a bunch of different places so we need them committed.

The workflow will do this but it's nice to be able to check the output in the PR too 👍

Comment (evoactivity) [reply syntaxes/src/markdown.glimmer.codeblock.mjs:1]: @lifeart just thought, if the output _is_ the same, it won't be able to be committed right? There will be no git diff and nothing to commit?

Comment (lifeart) [reply syntaxes/src/markdown.glimmer.codeblock.mjs:1]: @evoactivity yep, right, just waiting for formal confirm that it's ok :)

Comment (spuxx1701) [reply syntaxes/src/markdown.glimmer.codeblock.mjs:1]: I've been running the build script during development, but the output is the same, so nothing to commit. :)

Comment (evoactivity) [reply syntaxes/src/markdown.glimmer.codeblock.mjs:1]: Sounds like you've found your next contribution 😄

**Pull request merged**